policy/modules/admin/usbguard.fc

```diff
@@ -0,0 +1,7 @@
+/etc/usbguard					-d	gen_context(system_u:object_r:usbguard_conf_t,s0)
+/etc/usbguard/rules.conf				gen_context(system_u:object_r:usbguard_rules_t,s0)
+/etc/usbguard/.*					gen_context(system_u:object_r:usbguard_conf_t,s0)
```

Comment:
The lint fails on this regex. It should probably be `/etc/usbguard/.+`

Reply:
OK